.editorconfig

```diff
@@ -1,39 +1,24 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
 # WordPress Coding Standards
-# https://make.wordpress.org/core/handbook/best-practices/coding-standards/
+# https://developer.wordpress.org/coding-standards/wordpress-coding-standards/
 
 root = true
 
 [*]
 charset = utf-8
 end_of_line = lf
-indent_size = 4
-tab_width = 4
-indent_style = tab
 insert_final_newline = true
 trim_trailing_whitespace = true
+indent_style = tab
 
-[*.md]
-trim_trailing_whitespace = false
+[*.{yml,yaml}]
 indent_style = space
 indent_size = 2
```

Comment:
This is confusing to me. As far as I can tell, all the `.yml` files were now updated to use indent size 4, which looks right. But then why does this say indent size 2 here? 🤔